exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxy.java

```diff
@@ -86,9 +93,34 @@
       String name, View view, Serializer<K> keySerializer, Serializer<V> valueSerializer) {
     IndexAddress address = IndexAddress.valueOf(name);
     long viewNativeHandle = view.getViewNativeHandle();
-    LongSupplier nativeMapConstructor = () -> nativeCreate(name, viewNativeHandle);
+    LongSupplier nativeMapConstructor = () -> nativeCreate(name, viewNativeHandle, true);
+
+    return getOrCreate(address, view, keySerializer, valueSerializer, nativeMapConstructor, true);
+  }
+
+  /**
+   * Creates a ProofMapIndexProxy with no key hashing. Requires that keys are 32-byte long.
```

Comment:
Would it be possible to add a short warning with a link to the class-level section?